agent/agent.go

```diff
@@ -518,6 +500,15 @@ func (a *Agent) setupTask(agentCtx, setupCtx context.Context, tcInput *taskConte
 	return tc, setupFailed, shouldExit, nil
 }
 
+func (a *Agent) handleSetupError(err error, tc *taskContext, ctx context.Context, shouldSetLogger bool) (*taskContext, bool, bool, error) {
```

Comment:
I would name the output parameters since they're not obvious.

Reply:
it's now only one bool that's named when it's returned